.github/workflows/php.yml

```diff
@@ -20,6 +20,7 @@ jobs:
           - "7.4"
           - "8.0"
           - "8.1"
+          - "8.2"
```

Comment:
Just a thought, but why not automate this section with: https://github.com/WyriHaximus/github-action-composer-php-versions-in-range?tab=readme-ov-file#example